priv/template/Dockerfile.eex

```diff
@@ -17,9 +17,10 @@ RUN mkdir ../deps && mkdir ../_build
 
 # Build all dependencies separately to the application code
 RUN mkdir config
-COPY config/* config/
 COPY mix.* ./
-RUN mix do deps.get, deps.compile
+RUN mix do deps.get
+COPY config/* config/
```

Comment:
smart

Reply:
Yeah, I think this should work. 

Before I take the WIP label off I'll probably merge some docker layers for some extra optimised optimisation.